blog/authors.yml

```diff
@@ -0,0 +1,26 @@
+cloudposse:
+  name: Cloud Posse
+  title: Cloud Posse
+  url: https://cloudposse.com
+  image_url: https://avatars.githubusercontent.com/u/16398900?s=200&v=4
+milldr:
+  name: Daniel Miller
+  title: Cloud Posse
+  url: https://github.com/milldr
+  image_url: https://github.com/milldr.png
+osterman:
+  name: Erik Osterman
+  title: Cloud Posse
+  url: https://github.com/osterman
+  image_url: https://github.com/osterman.png
+goruha:
+  name: Igor Rodionov
+  title: Cloud Posse
+  url: https://github.com/goruha
+  image_url: https://github.com/goruha.png
+Benbentwo:
```

Comment:
```suggestion
benbentwo:
```

Reply:
this is the correct capitalization: https://github.com/Benbentwo